backend/controllers/adminControllers.js

```
@@ -0,0 +1,196 @@
+const mongoose = require('mongoose');
+const jwt = require("jsonwebtoken");
+
+
+const getUsers = async (req, res) => {
+  try {
+    const page = Number.parseInt(req.query.page) || 1;
+    const limit = Number.parseInt(req.query.limit) || 10;
+    const skip = (page - 1) * limit;
+    
+    const allowedRoles = ["admin", "vendor", "user"];
+    let filter = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'filter' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)